frontend/src/Commons/Forms/InputModules/FieldsAndSoil/FieldsAndSoil.tsx

```diff
@@ -178,7 +178,7 @@ const FieldsAndSoilComponent: FC<InputModuleProps> = ({
                     name="FieldName"
                     type="text"
                   />
-                  <StyledAreaContainer>
+                  <StyledAreaContainer formCrops={false}>
```

Comment:
What's up with this? Could you make `formCrops` optional so we don't need to pass this prop everywhere? Or is this necessary here?

Reply:
It's necesarry. Unless there's a way to make it default to false on emotion.